charts/nexus3/values.yaml

```diff
@@ -392,7 +392,8 @@ config:
     groupMemberFormat:
   # -- Blob store configuration.
   blobStores: []
-  # Reference the Nexus Blob store REST API for supported types and expected request body structures of each
+  # Reference the Nexus Blob store REST API (https://help.sonatype.com/en/blob-store-api.html) for supported types and expected request body structures of each
+  # You can also find the API reference on an existing Nexus installation at Administration section under System → API
```

Comment:
I don't think this adds much value, the comment above is where to make changes but I'm not aware of any useful publically available docs.

Reply:
It is the official URL for the Rest-API. Admittedly without much information (yet). Do you want me to remove it?

Reply:
If you think it's likely to get more comprehensive then I'm happy for it to be included, although I've not seen any progress towards this since I started working with the Nexus API many years ago. My main point is that if you want to improve the docs then you need to work with the helm-docs comments (the ones starting with `# -- `).

Reply:
The `--` wasn't there before. Anyway, I've added it and I think it's important to know where you can find documentation about the Rest API. An existing installation may be the best option here.